snowfox.yml

```diff
@@ -0,0 +1,227 @@
+version: '2'
+
+services:
+################################################################################
+#
+# The CAN Interfaces for Snowfox need an odsupercomponent.
+#
+################################################################################
+
+    odsupercomponent:
+        container_name: odsupercomponent
+        image: seresearch/opendlv-rhino-on-opendlv-on-opendlv-core-on-opendavinci-on-base:v0.1.5
+        restart: on-failure
+        network_mode: "host"
+        volumes:
+        - .:/opt/opendlv.data
+        cap_add:
+        - SYS_NICE
+        entrypoint: ""
+        command: "nice -n 0 /opt/od4/bin/odsupercomponent --configuration=/opt/opendlv.data/configuration --cid=110"
+
+######################################################################################
+#
+# Microservices for SNOWFOX (Volvo XC90).
+#
+######################################################################################
+   
+    # RTK Corrections for Applanix (needs update)
+#    dlv-ntrip:
```

Comment:
We did not test this, right?

Reply:
yes, we did not. It is not being used right now.